ripley/merged/roster.py

```diff
@@ -52,17 +47,22 @@ def _section(canvas_section):
 
 
 def _student(canvas_student, sections_by_id):
+    def _get(attr):
+        value = None
+        if hasattr(canvas_student, attr):
+            value = getattr(canvas_student, attr)
+        return value
     names = canvas_student.sortable_name.split(', ')
     enrollments = canvas_student.enrollments if hasattr(canvas_student, 'enrollments') else []
     return {
-        'email': canvas_student.email if hasattr(canvas_student, 'email') else None,
+        'email': _get('email'),
         'enrollStatus': enrollments[0]['enrollment_state'] if enrollments else None,
         'firstName': names[1],
         'gradeOption': 'TODO',
         'id': canvas_student.id,
         'lastName': names[0],
-        'loginId': canvas_student.login_id if hasattr(canvas_student, 'login_id') else None,
-        'photoUrl': 'TODO',
+        'loginId': _get('login_id'),
+        'photoUrl': _get('avatar_url'),
```

Comment:
@johncrossman I thought we'd be using an S3 URL pointing to the Cal 1 Card data. Is it ok to use the Canvas avatar instead? That would remove the need for even more overhead.

Reply:
@lyttam - You are right. I'll amend to remove it.